### ISSUE TITLE
[Bug]: Number of Files not updated

### ISSUE THREAD

**Issue by t71rs** [Stirling-Tools/Stirling-PDF#1547]
### The Problem

When using Add image to PDF or Sign PDF the second file input form is a liitle bit faulty. This is just a visual error, and does not impact functionality. In the picture below you can  see on the left a pdf where i added two images. On the right side i have deletedone of the images but it still says that i have selected two files.
![image](https://github.com/Stirling-Tools/Stirling-PDF/assets/166131967/d805dc84-f06e-4cf7-bb9e-c337c3ada127)

### Version of Stirling-PDF

0.24.6

### Additional Information

I have worked alot with the file input form in the past weeks, so I might be able to pick this up. But since i am not sure  I do not want to "claim" this :).

### No Duplicate of the Issue

- [X] I have verified that there are no existing issues raised related to my problem.

**Comment by Frooodle** [comment]
Yes right now this feature does not support multi PDFs, to be honest it shouldn't even allow it

**Comment by Frooodle** [comment]
Id prefer to disable multi file inputs for these features to avoid confusion

**Comment by Frooodle** [comment]
Old issue with old file selector, closing as not applicable
